### PR TITLE
Update seeds

### PR DESCRIPTION
**Pull request by jessieay**
* Calling `dev:prime[EMAIL]` will create a user for email and make it an
  admin
* This task clears out DB so there are no problems with uniqueness
* Creates all records necessary to preview emails

https://trello.com/c/voBTTwVx

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -4,6 +4,7 @@
 #
 MYUSA_KEY: consumer_public_key
 MYUSA_SECRET: consumer_secret_key
+DEFAULT_URL_HOST: your_local_server_address
```
Comment (pkarman) [inline .env.example]: is this really necessary to be set? should it not default to localhost?

Comment (jessieay) [inline .env.example]: we do need this for urls to work in emails -- see `config/initializers/mailer.rb` -- this is a newish change

Comment (pkarman) [inline .env.example]: then maybe it should default to something working? `localhost` ?

Comment (jessieay) [inline .env.example]: sounds good! question: are you using foreman or rails server? I am using foreman bc of our multiple processes but I see a lot of localhost:3000 in setup so maybe I should just default to that...

Comment (pkarman) [inline .env.example]: I use both, but mostly `rails s`

Comment (pkarman) [inline .env.example]: fwiw, I think default host is just hostname sans port.

Comment (jessieay) [inline .env.example]: I have it set to `http://localhost:5000` locally and that is working for me . http://guides.rubyonrails.org/action_mailer_basics.html#generating-urls-in-action-mailer-views

Comment (jessieay) [inline .env.example]: updated at 31ebada

File: .env.example [hunk 2]
```diff
@@ -4,7 +4,7 @@
 #
 MYUSA_KEY: consumer_public_key
 MYUSA_SECRET: consumer_secret_key
-DEFAULT_URL_HOST: your_local_server_address
+DEFAULT_URL_HOST: "http://localhost:3000"
```
Comment (pkarman) [inline .env.example]: I think this should just be `localhost`

**Comment by pkarman** [comment]
@jessieay when I check out how `DEFAULT_URL_HOST` is used at cloud.gov, it is like this:

`DEFAULT_URL_HOST: cap.18f.gov`

which suggests it should just be `localhost` (or maybe `localhost:3000`) but definitely no protocol.

**Comment by jessieay** [comment]
@pkarman updated at fc4463b

**Pull request merged**